extensions/yaml_support/labgraph_monitor/server/lg_monitor_server.py

```diff
@@ -70,14 +53,12 @@ def setup(self) -> None:
             self.WS_SERVER_NODE.configure(wsapi_server_config)
 
         def connections(self) -> lg.Connections:
-            return (
-                # Test Node Connections
-                (self.NOISE_GENERATOR.OUTPUT, self.SERIALIZER.INPUT),
-                (self.SERIALIZER.TOPIC, self.WS_SERVER_NODE.topic),)
+            local = ((self.SERIALIZER.TOPIC, self.WS_SERVER_NODE.topic),)
+            tuples = local + graph.connections()
+            return tuples
```

Comment:
I then set up connections both from the original `WSSender`, `((self.SERIALIZER.TOPIC, self.WS_SERVER_NODE.topic),)`
**and from the `Demo` graph**, using `graph.connections()`